### PR TITLE
Clintonb/ecommerce update rebase

### PR DESCRIPTION
**Pull request by staubina**
This is simply a PR to highlight the changes made to get the ecommerce docker image working. I will work with @clintonb to integrate these changes.

### CODE REVIEW

File: playbooks/roles/ecommerce/defaults/main.yml [hunk 1]
```diff
@@ -44,8 +44,11 @@ ECOMMERCE_JWT_ALGORITHM: 'HS256'
 ECOMMERCE_JWT_VERIFY_EXPIRATION: true
 ECOMMERCE_JWT_DECODE_HANDLER: 'ecommerce.extensions.api.handlers.jwt_decode_handler'
 ECOMMERCE_JWT_ISSUERS:
-  - '{{ COMMON_JWT_ISSUER }}'
-  - 'ecommerce_worker'  # Must match the value of JWT_ISSUER configured for the ecommerce worker.
```
Comment (staubina) [inline playbooks/roles/ecommerce/defaults/main.yml:63]: We need to update this and add an issuer for ecommerce_worker

File: playbooks/roles/ecommerce/tasks/main.yml [hunk 2]
```diff
@@ -10,22 +10,23 @@
...
 
-- name: Create demo data
-  shell: '{{ ecommerce_venv_dir }}/bin/python ./manage.py create_demo_data --partner=edX'
-  args:
-    chdir: "{{ ecommerce_code_dir }}"
-  become_user: "{{ ecommerce_user }}"
-  environment: "{{ ecommerce_environment }}"
-  when: ecommerce_create_demo_data
-  tags:
-    - install
```
Comment (staubina) [inline playbooks/roles/ecommerce/tasks/main.yml]: Change this from install to migrate and/or migrate:db

Comment (clintonb) [reply playbooks/roles/ecommerce/tasks/main.yml]: Setting `ecommerce_create_demo_data` in https://github.com/edx/configuration/blob/master/docker/build/ecommerce/ansible_overrides.yml ensures these tasks don't get run for Docker builds.

File: playbooks/roles/ecommerce/defaults/main.yml (hunk 1 above)
Comment (clintonb) [reply playbooks/roles/ecommerce/defaults/main.yml:63]: I think your rebase got rid of my override of `ECOMMERCE_JWT_ISSUERS`.

Comment (staubina) [reply playbooks/roles/ecommerce/defaults/main.yml:63]: I think we should probably throw this branch away and focus on your branch now that you have rebased.

Comment (clintonb) [reply playbooks/roles/ecommerce/defaults/main.yml:63]: Agreed.

**Comment by staubina** [comment]
@clintonb Should I close this PR?

Have you had a chance to deploy a new version of the ecommerce image with the devstack tag?

**Comment by clintonb** [comment]
I have not. Keep this open for now. I'll close once my work is done/merged.

**Comment by clintonb** [comment]
#3941 merged

**Pull request closed without merging**